.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,49 @@
+name: CI Workflow
+
+on:
+  [push, workflow_dispatch]
+
+concurrency: ci-${{ github.ref }}
+
+jobs:
+  buildAndTest:
+    name: Build, Test, and Package
+    runs-on: ubuntu-latest
+    env:
+      RESOURCE_GROUP: hri-dev1-wdc-kube
```

Comment:
Nit -- a newline after the env vars would be good

Reply:
+1 for Dan's Comment. Agree.